packages/grafana-ui/src/types/panel.ts

```diff
@@ -9,6 +11,7 @@ export interface PanelProps<T = any> {
   renderCounter: number;
   width: number;
   height: number;
+  onInterpolate: InterpolateFunction;
```

Comment:
Is `onInterpolate` required prop for all panels?

Reply:
Good question, it might not be as this is the interface for our external plugin as well. What do @torkelo think?

Reply:
not sure what you mean required prop, it's a prop provided by Grafana to all panels to interpolate string. 

It's not something all panels will use